manifests/istio-control/istio-discovery/values.yaml

```diff
@@ -153,6 +153,10 @@ telemetry:
   v1:
     # Set true to enable Mixer based telemetry
     enabled: true
+    # Set true to enable Mixer for TCP traffic
+    # Superseeded by the above telemtry.v1.enabled.
+    tcp:
```

Comment:
v2 enabled already implies tcp v1 I think?

Reply:
Adding this option, so that users can go back to behavior of 1.4

In 1.5(pending approval)=>
v2.enabled => tcp and http v2
if user wants to go back to 1.4 behavior:
v1.tcp.enabled => tcp v1 and http v2

Reply:
What if they set v1.enabled and v2.enabled? Today this is how they get `tcp v1 and http v2`

Reply:
oh ok.. in that case we dont need a new value

we can say as follows:
v2.enabled,  v1.disabled : v2 http,  v2 tcp
v2.enabled, v1.enabled: v2 http, v1 tcp
v2.disabled, v1.disabled: no telemetry
v2.disabled, v1.enabled: v1 http, v1 tcp

@mandarjog , @bianpengyuan : what do you guys think too?

Reply:
I am ok with the above, but also open to other ideas